bus/bus.go

```diff
@@ -92,7 +92,8 @@ type (
 		SetContractSet(ctx context.Context, set string, contracts []types.FileContractID) error
 
 		Object(ctx context.Context, key string) (object.Object, error)
-		Objects(ctx context.Context, key string) ([]string, error)
+		Objects(ctx context.Context, offset, limit int, key, prefix string) ([]string, error)
```

Comment:
`nit` can we reserve the order of the parameters here, makes more sense if you specify what to search for followed by an offset and then limit